### PR TITLE
Upgrade lashup make faster

### PR DESCRIPTION
**Pull request by sargun**

### CODE REVIEW

**Comment by sargun** [comment]
@GoelDeepak please review.

**Comment by GoelDeepak** [comment]
@Sargun Could you please have a comment summarizing this patch?

**Comment by sargun** [comment]
* Gets rid of SRV record generation
* Moves to a faster Lashup

File: apps/navstar_dns/src/navstar_dns_poll_server.erl [hunk 1]
```diff
@@ -578,9 +443,13 @@ base_records(ZoneName) ->
 ops(OldRecords, NewRecords) ->
     RecordsToDelete = ordsets:subtract(OldRecords, NewRecords),
     RecordsToAdd = ordsets:subtract(NewRecords, OldRecords),
-    Ops0 = [],
-    Ops1 = lists:foldl(fun delete_op/2, Ops0, RecordsToDelete),
-    lists:foldl(fun add_op/2, Ops1, RecordsToAdd).
+    Ops0 = lists:foldl(fun delete_op/2, [], RecordsToDelete),
+    case RecordsToAdd of
+        [] ->
+            [];
```
Comment (GoelDeepak) [inline apps/navstar_dns/src/navstar_dns_poll_server.erl]: In case RecordsToAdd is [], Wouldn't the return be Ops0?

**Comment by GoelDeepak** [comment]
lgtm

**Pull request merged**